packages/ui/src/components/Breadcrumb/index.module.scss

```diff
@@ -16,6 +16,10 @@
   margin-inline-end: 8px;
 }
 
+.utrecht-breadcrumb-nav__separator {
+  margin-inline-end: 8px;
+}
```

Comment:
is er geen design token voor, min width van de separator ofzo?

Reply:
goed nieuws die margin was blijkbaar niet nodig

Reply:
er is geen width token ik kan wel de margin invullen met een spacing token